src/lib/ext-listen.js

```diff
@@ -31,3 +31,19 @@ export function openActivityLogPage() {
     url: getActivityLogPageURL(),
   });
 }
+
+export function dateTimeFormat(timestamp, options) {
+  const dateTime = new Date(timestamp);
+  const time = dateTime.toLocaleTimeString();
```

Comment:
`DateTimeFormat` also supports parameters to include the time in the generated string. Do not use `toLocaleTimeString`, but only `DateTimeFormat`.